image-download

```diff
@@ -54,6 +54,11 @@ CONFIG = "~/.config/image-stores"
 DEVNULL = open("/dev/null", "r+")
 EPOCH = "Thu, 1 Jan 1970 00:00:00 GMT"
 
+if os.isatty(1):
+    SYMBOLS = {'present': ' ✔', 'absent': ' ⨯', 'selected': '❯❯'}
```

Comment:
have to admit I'm a bit sad to see the snail go :rofl:

Reply:
As I said I'm perfectly happy to leave the penalty thing as an imperfect solution for now, until we figure out something better. I started to work on parallel downloads and such, but it will look rather messy.